config/jobs/kubernetes-sigs/service-catalog/service-catalog-presubmits.yaml

```diff
@@ -1,3 +1,12 @@
+build_image_cfg: &build_image_cfg
+  decorate: true
+  decoration_config:
```

Comment:
Using YAML anchors is contentious, but I think it's appropriate for these defaults for this file.

However, I think it should be applied to `pull-service-catalog-integration` as well, if it's going to be used.

Reply:
Why is it considered contentious? I will admit I don't have a lot of experience with setting up stuff on Prow. 

Either way I agree we should be consistent.

Reply:
It's not about the prow itself but about writing YAML files. If there is an agreement that using anchors are restricted then I can revert that. I've just used that to DRY but I know that it can sometimes obfuscate the code :)